commit 67bc278a511ca6ec42e1f8d2d5897c0109e94e2c
Author: Dan Nicholson <dbn.lists@gmail.com>
Date:   Wed Feb 10 15:36:44 2010 +1000

    xfree86: Make InputClass docs and comments match reality
    
    Drivers and options specified in InputClass sections work on a "first
    match wins" strategy. Let's be consistent when documenting it.
    
    Signed-off-by: Dan Nicholson <dbn.lists@gmail.com>
    Reviewed-by: Peter Hutterer <peter.hutterer@who-t.net>
    Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>

Index: xorg-server/hw/xfree86/common/xf86Xinput.c
===================================================================
--- xorg-server.orig/hw/xfree86/common/xf86Xinput.c	2010-03-24 13:24:59.000000000 +0200
+++ xorg-server/hw/xfree86/common/xf86Xinput.c	2010-03-24 13:25:06.000000000 +0200
@@ -540,9 +540,9 @@
 }
 
 /*
- * Merge in any InputClass configurations. Each InputClass section can
- * add to the original device configuration as well as any previous
- * InputClass sections.
+ * Merge in any InputClass configurations. Options in each InputClass
+ * section have less priority than the original device configuration as
+ * well as any previous InputClass sections.
  */
 static int
 MergeInputClasses(IDevPtr idev, InputAttributes *attrs)
Index: xorg-server/hw/xfree86/doc/man/xorg.conf.man.pre
===================================================================
--- xorg-server.orig/hw/xfree86/doc/man/xorg.conf.man.pre	2010-03-24 13:24:59.000000000 +0200
+++ xorg-server/hw/xfree86/doc/man/xorg.conf.man.pre	2010-03-24 13:25:06.000000000 +0200
@@ -1020,7 +1020,7 @@
 entry specifies the name of the driver to use for this input device.
 After all classes have been examined, the
 .RI \*q inputdriver \*q
-module from the final
+module from the first
 .B Driver
 entry will be enabled when using the loadable server.
 .PP
